ref/general/password-encryption.adoc

```diff
@@ -17,21 +17,26 @@ Open Liberty supports AES encryption for passwords that are stored in the `serve
 When you use this option for protecting system passwords in the Open Liberty configuration, you need to understand the limits to the protection it provides.
 For information about how to obfuscate passwords for Open Liberty, see the link:docs/ref/general/securityUtility.adoc[`securityUtility` command].
 
+== Password encryption with Open Liberty
+
 Encrypting a password in the Open Liberty configuration doesn’t guarantee that the password is secure or protected.
-It only means that someone who can see the encrypted password but doesn’t know the encryption key, cannot easily recover the password.
+It means that someone who can see the encrypted password but doesn’t know the encryption key cannot easily recover the password.
+
 The application server process requires access to both the encrypted password and the decryption key, so both these items need to be stored on the file system that is accessible to the server runtime environment.
 The encryption key is also required by anyone who encrypts a password that is placed in the server configuration.
-For an attacker that has access to exactly the same set of files as the Open Liberty server instance, applying AES encryption to the password provides no additional security over XOR encoding.
+For an attacker that has access to the same set of files as the Open Liberty server instance, applying AES encryption to the password provides no additional security over XOR encoding.
 
 However, there are still reasons why you might consider encrypting passwords in the Open Liberty configuration.
-The Open Liberty configuration is designed to be highly composable and sharable.
-As such, it’s designed to be configured without an administration subsystem, so any XOR-encoded password is visible to any administrator.
-Given these design features, consider the following scenarios:
+The Open Liberty configuration is highly composable and sharable.
```

Comment:
I think you could remove the word "highly" here as it may not add value to the statement without additional context. In general, I think it is usually best to avoid qualitative statements that may not be verifiable without a point of comparison or additional information.